pvlib/test/test_irradiance.py

```diff
@@ -365,3 +368,29 @@ def test_erbs_all_scalar():
 
     for k, v in out.items():
         assert_allclose(v, expected[k], 5)
+
+
+def test_dirindex():
+    clearsky_data = tus.get_clearsky(times, model='ineichen',
+                                     linke_turbidity=3)
+    ghi = pd.Series([0, 0, 1038.62, 254.53], index=times)
+    ghi_clearsky = clearsky_data['ghi']
+    dni_clearsky = clearsky_data['dni']
+    zenith = ephem_data['apparent_zenith']
```

Comment:
Your test is going to be more robust you hard code these input values. The values themselves are fine, it's only that they're derived at runtime from functions that could produce slightly different results in the future and thus might make your new test fail for no good reason. You'll can see that the inputs are precisely defined in the dirint tests that actually check values.